src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -81,11 +89,7 @@ public void initialize() {
                 (newSelectedGroups) -> newSelectedGroups.forEach(this::selectNode);
         Consumer<List<TreeItem<GroupNodeViewModel>>> updateViewModel =
```

Comment:
With these changes, I think the `updateViewModel` consumer can be replaced by `this::updateSelection` (or whatever the exact syntax is for these generics)

Reply:
Ok, I'll have a check and see what I can do.

Reply:
Done